### ISSUE TITLE
Disable Proposed Sensors features in UI

### ISSUE THREAD

**Issue by HansVraga** [USGS-WiM/STNWeb#425]
From Todd:
>Hans,

> 
> The Proposed Sensors feature is not completely built out yet, if I remember correctly, and I'm getting some problems with people using that feature and then getting some database error caused by the proposed sensor, even after you delete the proposed sensor.  For example, when trying to delete site ALMOB24771, I get the following error:
> 
> ![image](https://user-images.githubusercontent.com/12058013/38515936-bfc50d94-3c03-11e8-9e18-df31a150ac9e.png)
> 
> 
> I believe this is caused by the previous existence of a Proposed sensor.
> 
> **Can we get the Proposed Sensors section removed from the GUI for now until that feature is finished?**
> 
> -Todd

**Comment by lprivette** [comment]
I'm working on hiding/commenting out the proposed sensors.

It might be useful to get some steps to reproduce from Todd. I can't seem to recreate it, but I've definitely seen that error before (can't remember what for).

**Comment by tkoenigusgs** [comment]
I just realized that it may not be the Proposed sensors that threw the above error.  That error may have been thrown by the presence of Networks being listed in the site.  Once the network names were deleted from the Site info, the site could be deleted without error.  I'm sure the Proposed Sensors are still causing us problems elsewhere, so I'm in favor of removing Proposed Sensors from the production system until we get it built out.  But we may be able to close this particular issue once the Proposed Sensor removal is complete for production.  I'll ask Hans to open a new issue that relates the error above with Network names in the site to be deleted.

**Comment by tkoenigusgs** [comment]
Looks good in test.  Looking forward to seeing it in production.  -Todd
